modules/invoicein/model/invoicein.class.php

```diff
@@ -479,6 +477,26 @@ public function journal() {
                         $VoucherH['voucher_Description']    = '';
                         $VoucherH['voucher_AccountPlanID']  = 0;
 
+                        //#Motkonto resultat.
+                        $VoucherH['voucher_AccountPlanID']  = $line->AccountPlanID;
+                        if($line->IsOnlyTax) {
+                            $VoucherH['voucher_AccountPlanID'] = 2710;
+                        }
+                        $line_accountplan = accounting::get_accountplan_object($VoucherH['voucher_AccountPlanID']);
```

Comment:
move to each separate else block so we don't do the setting of default if not necessary.